src/main/webapp/app/markdown-editor/markdown-editor.component.html

```diff
@@ -0,0 +1,26 @@
+<div>
+    <div class="tool-bar">
+        <div class="btn-group">
+            <ng-container *ngFor="let command of commands">
+                <button class="btn btn-sm btn-default" type="button" [title]="command.buttonTitle"
+                        (click)="command.execute(aceEditorContainer.getEditor())">
+                    <i [ngClass]="command.buttonIcon"></i>
+                </button>
+            </ng-container>
+                <div class="btn btn-outline-secondary"
```

Comment:
Why are we adding the Commands like that? They should be part of the `commands` Array!